libs/common/src/services/vault-timeout/vault-timeout-settings.service.ts

```diff
@@ -184,7 +293,8 @@ export class VaultTimeoutSettingsService implements VaultTimeoutSettingsServiceA
       if (decryptionOptions?.hasMasterPassword != undefined) {
         return decryptionOptions.hasMasterPassword;
       }
```

Comment:
I know I wrote this, but can you change it to be:
```suggestion
      !!decryptionOptions.hasMasterPassword
```
The else you added below means that we'd return undefined.

Reply:
Ah, good catch. Done!